platform/linuxbsd/x11/display_server_x11.cpp

```diff
@@ -2891,10 +2891,15 @@ void DisplayServerX11::window_move_to_foreground(WindowID p_window) {
 	XFlush(x11_display);
 }
 
+DisplayServerX11::WindowID DisplayServerX11::get_focused_window() const {
+	return last_focused_window;
```

Comment:
Is it intentional, that in this case `DisplayServerX11::_get_focused_window_or_popup()` is not used?

Reply:
its kinda not useful for popups, and popups can complicate things because this code could be called from a popup that is about to be closed, hence you get the wrong window.